### ISSUE TITLE
Bootstrap 4 label migrations

### ISSUE THREAD

**Issue by nbargnesi** [kevoreilly/CAPEv2#403]
# Prerequisites

- [ ] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (for multi-repository projects)

# Expected Behavior

Bootstrap badges render correctly in the web UI.

# Current Behavior

A handful of Bootstrap badges still use the legacy "label" class from v3 so they don't render properly.

Here's an example. This super bad sample should show with the danger badge:

<img width="380" alt="image" src="https://user-images.githubusercontent.com/1265294/103700383-5ff9bb00-4f72-11eb-9cb6-a85e49c91f5a.png">

# Failure Information (for bugs)

This was introduced when #353 was merged. The remaining classes need to be migrated from the "label" counterparts.

<img width="380" alt="image" src="https://user-images.githubusercontent.com/1265294/103700526-96373a80-4f72-11eb-839e-3af80da3efc9.png">

So they render properly:

<img width="380" alt="image" src="https://user-images.githubusercontent.com/1265294/103700542-9fc0a280-4f72-11eb-8889-2daf76184b3c.png">

## Steps to Reproduce

Visit any of the pages where the old labels are still used.

## Context

This is using the latest commit. I'll have a PR fix up in a minute.

## Failure Logs

NA

**Comment by doomedraven** [comment]
hello, thanks for PR, i suppouse we can close this as you submitted PR no? Thank you for PR 
well that wasn't fixed bcz we don't respect malscore is super useless

**Comment by nbargnesi** [comment]
> hello, thanks for PR, i suppouse we can close this as you submitted PR no?

Yes, I created the issue just to have something to reference in the pull request.

> that wasn't fixed bcz we don't respect malscore is super useless

Agreed. FWIW commit 652ff0d7 hits more than just malscore (MISP, procmemory, etc.). I only used malscore to demo the behavior.

**Comment by doomedraven** [comment]
you can provide the same details in PR, if that speedup everything
yes i saw, not sure how I missed others. thanks for fixing it
